tests/mock_cloud_sdk.py

```diff
@@ -0,0 +1,79 @@
+import json
+
+from pasqal_cloud.job import CreateJob
+from pasqal_cloud.device import BaseConfig, EmulatorType
+from pasqal_cloud.batch import Batch
+from pasqal_cloud.utils.responses import PaginatedResponse
+
+from uuid import uuid4
+
+from unittest.mock import MagicMock
+from typing import Any
+
+
+class MockSDK:
+    """Helper class to mock the cloud SDK and skip the API calls.
+
+    Warning: This implements only a small subset of the functions available
+        in the cloud SDK; focusing on the functions used by qek.
+        This should be extended to support all methods and moved to the
+        pasqal-cloud repository so that it can be reused for all future
+        libraries using the SDK.
```

Comment:
:+1:

Reply:
@MatthieuMoreau0 Could this be provided in the `pasqal_cloud` package ? I mean it'd be super useful in a number of places if we could get hold of a proper and exhaustive mock SDK.

Reply:
Yes that's the plan. As discussed with David, I suggest this lives in the qek repository for a while to make iterations quicker. Once we are happy iwth the mock we'll port it to the pasqal_cloud SDK so that it can be extended and reused.